src/System.Ben.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
```

Comment:
Shouldn't you also delete the empty lines, to further reduce the files size (also applies line 6)